run.sh

```diff
@@ -12,21 +12,89 @@ if [[ $(basename "${CONDA_PREFIX}") != "code_as_data" ]]; then
   exit 1
 fi
 
+###############################################################################
+
+# Helper functions
+
+function yaml_value {
+    cat "${1}" | shyaml get-value "${2}"
+}
+
+###############################################################################
+
+# Input data for the project
+task_view_url="$(yaml_value "${config_file}" "task_view_url")"
+
+# Files constructed by the called scripts
+cran_table="$(yaml_value "${config_file}" "cran_details_file")"
+repo_details="$(yaml_value "${config_file}" "repo_details_file")"
+dupree_benchmarks="$(yaml_value "${config_file}" "all_pkg_benchmarks_file")"
+cloc_table="$(yaml_value "${config_file}" "all_pkg_cloc_file")"
+gitsum_table="$(yaml_value "${config_file}" "all_pkg_gitsum_file")"
+
+# Parent directories for package-specific:
+# - local-storage of the github repos
+repo_dir="$(yaml_value "${config_file}" "repo_dir")"
+# - results
+pkg_results_dir="$(yaml_value "${config_file}" "pkg_results_dir")"
+
+###############################################################################
+
 # Run each step of the analysis:
 
-Rscript "scripts/01-get-devtools-cran-table.R" --config "${config_file}"
+Rscript "scripts/01-get-devtools-cran-table.R" \
+  --config "${config_file}" \
+  --url "${task_view_url}" \
+  --output "${cran_table}"
+
+Rscript "scripts/02-github-details.R" \
+  --input "${cran_table}" \
+  --repo_dir "${repo_dir}" \
+  --output "${repo_details}"
+
```

Comment:
Make temp file containing all package names here. This could be passed into the collapsing scripts, rather than the config script